package.json

```diff
@@ -14,20 +14,21 @@
   ],
   "scripts": {
     "build": "h2-build && turbo run build --filter='./apps/*' --filter='!@gc-digital-talent/e2e'",
+    "build:fresh": "npm run build -- --force --no-cache",
     "build:all": "turbo run build",
     "build:packages": "turbo run build --filter='./packages/*'",
     "build:css": "h2-build",
```

Comment:
I don't think so since we are running a script defined in a dependency and not in one of our `apps/*` or `packages/*`. But my confidence on that is not incredibly high either 😅